examples/mesos/README.md

```diff
@@ -0,0 +1,14 @@
+# Launching Weave Scope on Apache Mesos Cluster
```

Comment:
Yeah :)

On Thu, 26 May 2016, 09:18 Viktor Sadovnikov, <notifications@github.com>
wrote:

> In examples/mesos/README.md
> <https://github.com/weaveworks/scope/pull/1509#discussion_r64706931>:
>
> > @@ -0,0 +1,14 @@
> > +# Launching Weave Scope on Apache Mesos Cluster
>
> Are you suggesting adding minimesos to site/installing.md as a platform
> along with k8, DC/OS and Amazon?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/scope/pull/1509/files/58f80920b3a576939721f620dec6a4f6a8009a04#r64706931>
>